Networking/Networking/Remote/ProductsRemote.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import Alamofire
+
+
+/// Product: Remote Endpoints
+///
+public class ProductsRemote: Remote {
```

Comment:
I think we are going to want the ability to fetch a single product by id ([much like we do for orders](https://github.com/woocommerce/woocommerce-ios/blob/develop/Networking/Networking/Remote/OrdersRemote.swift#L44)).